Apps/W1/EDocument/app/src/Processing/Import/FinishDraft/EDocCreatePurchCrMemo.Enum.al

```diff
@@ -0,0 +1,18 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.eServices.EDocument.Processing.Import;
+
+using Microsoft.eServices.EDocument.Processing.Interfaces;
+
+/// <summary>
+///  Enum for the implementations of the E-Doc. Create Purchase Credit Memo interface.
+/// </summary>
+enum 6118 "E-Doc. Create Purch. Cr. Memo" implements IEDocumentCreatePurchaseCreditMemo
```

Comment:
Can we put the two create enums together into Create BC Docment, and then have 1 entry for invoice, and 1 entry for credit note? 

That would mean to obsolete the E-Doc. Create Purch. Invoice enum, which is fine. 
Otherwise for each document type we would have a new enum, which is not really needed.